LINK/usr/bin/evm_watchdog.rb

```diff
@@ -2,7 +2,8 @@
 # description: ManageIQ watchdog application loop
 #
 
-require '/var/www/miq/vmdb/gems/pending/util/system/evm_watchdog'
+require 'manageiq-gems-pending'
+require 'system/evm_watchdog'
```

Comment:
UGH, rubygems magic.  I know why we do it but I'd like to get away from relying on non-standard gemspec require paths.  I don't know, maybe your suggestion of `require 'util/system/evm_watchdog' ` is better.